### DISCUSSION TITLE
Aura theme

### DISCUSSION THREAD

**Discussion by gregogun** [ora-ui/ora-ui#9] [General]
As the foundation of the design system, it's important that the decisions made about the theme are well-informed and any and all opinions justified. 

The point of posting this here is really just to generate discussion around what may be the best way to approach building design tokens that can be adopted and customized by various teams with different needs and visual languages. 

The current implementation can be found [here](https://github.com/aura-ui/aura-ui/blob/be771cbbae98c5afe79390b39262b7197d7faece/packages/theme/stitches.config.ts), and as you might notice is largely inspired by the tailwind theme. This of course has the benefit of increased adoption in the dev community as many devs are familiar with tailwind.

Keen to hear thoughts on this. and if there might be a better way to approach this :)

**Comment by Code-Victor** [comment]
Using  `P` for padding is somewhat problematic
![image](https://user-images.githubusercontent.com/63613078/205293074-9825616d-d116-49d2-a7a8-7b297eaea89c.png)
I came across this problem while trying to style a `<p>` element in stitches. There would be a name collision.
```tsx
const BlogContainer = styled("div", {
  bg: "$primary",
  px: "$4",
  py: "$2",
  p: {
    color: "$secondary",
    fontSize: "$2",
    lineHeight: "$3",
  },
});
```
My solution was to use `pd` instead of `p` for my padding.

**Reply by gregogun** [reply]
Hmm interesting, I've never come across this problem. 

I think it's because when styling any html element inside a styled function I'll usually do it like this 
```
  '& p': {
    color: "$secondary",
    fontSize: "$2",
    lineHeight: "$3",
  }
  ```
  
But in any case I'd be happy to add in `pd` additionally to prevent that issue 👍

Thanks for your input!
